forge-gui/res/cardsfolder/o/onakke_oathkeeper.txt

```diff
@@ -0,0 +1,8 @@
+Name:Onakke Oathkeeper
+ManaCost:1 W
+Types:Creature Ogre Spirit
+PT:0/4
+S:Mode$ CantAttackUnless | ValidCard$ Creature | Target$ Planeswalker.YouCtrl | Cost$ 1 | Description$ Creatures can't attack planeswalkers you control unless their controller pays {1} for each creature they control that’s attacking a planeswalker you control.
```

Comment:
noncurly versions please
’ = '

Reply:
look closely depending on your font, the apostrophes are not the same
in fact you already mixed them in that description
for consistency we only use the simple version (not sure if some older skins still behave better with ASCII or we can eventually do a big update)

Reply:
Ah okay thank you, fixed :)